test/go-tests/test_qualitygate.go

```diff
@@ -96,16 +96,12 @@ func Test_QualityGates(t *testing.T) {
 
 	source := "golang-test"
 
-	resp, err := ExecuteCommand(fmt.Sprintf("kubectl get configmap -n %s lighthouse-config-%s", GetKeptnNameSpaceFromEnv(), projectName))
-	if !strings.Contains(resp, "not found") && err != nil {
-		_, err = ExecuteCommand(fmt.Sprintf("kubectl delete configmap -n %s lighthouse-config-%s", GetKeptnNameSpaceFromEnv(), projectName))
-		require.Nil(t, err)
-	}
-	t.Logf("creating project %s", projectName)
-
 	projectName, err = CreateProject(projectName, shipyardFilePath, true)
 	require.Nil(t, err)
 
+	_, _ = ExecuteCommand(fmt.Sprintf("kubectl delete configmap -n %s lighthouse-config-%s", GetKeptnNameSpaceFromEnv(), projectName))
```

Comment:
I would also add a log output above this line to be consistent and have a good logging